### PR TITLE
hello-world

### PR DESCRIPTION
**Pull request by djodjo89**

### CODE REVIEW

**Review by Adham-A — COMMENTED** [review]
Simple or double quotes that's the question...

**Review by alexandrekd — COMMENTED** [review]
pas mal

**Review by indyteo — COMMENTED** [review]
Coucou

**Review by Cedric-F — APPROVED** [review]
I prefer singles, of course.

File: index.js [hunk 1]
```diff
@@ -0,0 +1 @@
+console.log('Hello World!');
```
Comment (Adham-A) [inline index.js:1]:
```suggestion
console.log("Hello World!");
```

**Review by indyteo — APPROVED** [review]

**Review by alexandrekd — APPROVED** [review]

**Review by Adham-A — APPROVED** [review]

**Comment by Adham-A** [comment]
This comment will die with this branch.

**Pull request merged**